uproot/tree.py

```diff
@@ -1816,17 +1816,17 @@ def _init(self):
             self.keycache = cachetools.LRUCache(10000)                      # last 10000 TKeys
 
     def __getstate__(self):
-        return {"paths": paths,
-                "treepath": treepath,
-                "branches": branches,
-                "entrysteps": entrysteps,
-                "flatten": flatten,
-                "awkwardlib": awkwardlib,
-                "persistvirtual": persistvirtual,
-                "localsource": localsource,
-                "xrootdsource": xrootdsource,
-                "httpsource": httpsource,
-                "options": options}
+        return {"paths": self.paths,
+                "treepath": self.treepath,
+                "branches": self.branches,
+                "entrysteps": self.entrysteps,
+                "flatten": self.flatten,
+                "awkwardlib": self.awkwardlib,
+                "persistvirtual": self.persistvirtual,
+                "localsource": self.localsource,
+                "xrootdsource": self.xrootdsource,
+                "httpsource": self.httpsource,
+                "options": self.options}
```

Comment:
I wonder how this happened... someone must have asked for `_LazyFiles` to be pickleable, and I added this half-hearted attempt without checking it? Thanks for fixing it.